build/pion-boost.inc

```diff
@@ -134,25 +134,22 @@ if test "x$enable_tests" == "xno"; then
 	# Display notice if unit tests are disabled
 	AC_MSG_NOTICE([Unit tests are disabled])
 else
-	BOOST_TRY_LIB=unit_test_framework
-	BOOST_TRY_LINK="$BOOST_HOME_DIR/lib/libboost_${BOOST_TRY_LIB}${BOOST_LIB_EXTENSION}.a"
 	# check if static version of lib is available
-	if !(test -r "$BOOST_TRY_LINK"); then
```

Comment:
@mikedickey Well I'm not sure if it's the right solution - but the issue is that on Debian you get both the static `.a` and the dynamic `.so` installed, but this code will prefer to link the `.a` file which causes problems for applications that link dynamically boost elsewhere.